trace-sdk/src/test/java/io/bitrise/trace/data/management/formatter/BaseDataFormatter.java

```diff
@@ -10,7 +10,7 @@
  * This ensures the ApplicationSessionManager is started before which is required to be able to
  * format any data
```

Comment:
nit: I just spotted that there is no line ending dot, could you please add it there?

Reply:
Good catch. Added :)